packages/inter-protocol/test/smartWallet/contexts.js

```diff
@@ -52,14 +52,18 @@ export const makeDefaultTestContext = async (t, makeSpace) => {
     { storageNode, walletBridgeManager },
   );
 
-  const simpleProvideWallet = async address => {
+  const provideBankAndWallet = async address => {
     // copied from makeClientBanks()
-    const bank = E(consume.bankManager).getBankForAddress(address);
+    const bank = await E(consume.bankManager).getBankForAddress(address);
 
     const [wallet, _isNew] = await E(
       walletFactory.creatorFacet,
     ).provideSmartWallet(address, bank, consume.namesByAddressAdmin);
-    return wallet;
+    return { bank, wallet };
```

Comment:
reminder to not expose this to the test script. e.g. provide a balance querier instead

Reply:
oh... that's still TODO...